go/felix/proto/felixbackend.proto

```diff
@@ -258,6 +259,7 @@ message HostEndpoint {
   string name = 1;
   repeated string profile_ids = 2;
   repeated TierInfo tiers = 3;
+  repeated TierInfo untracked_tiers = 6;
```

Comment:
Yes.  That's the correct way to evolve a protobuf schema.  You shouldn't change the IDs assigned to existing fields if you want to maintain back-compat.  However, you can put the new field in any position in the definition in order to maximise readability.